src/App.tsx

```diff
@@ -26,70 +27,132 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort();
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort((a, b) => a.length - b.length);
+      break;
+
+    default:
+  }
```

Comment:
It looks like you forgot to write the default case.

Reply:
I tried to add something but i got a diffirent error each time

Reply:
Have you tried `return 0` ?

Reply:
I think it's better to write `switch` inside the `sort` method, you will not repeat code with sort several times and you can make a normal default case like `return 0`.

Reply:
Return 0 returns an error, switch inside sort sound good